src/kschmaus1.java

```diff
@@ -0,0 +1,175 @@
+import java.util.*;
+import java.lang.*;
+
+class kschmaus1 {
+	private enum Operator {
+		ADD(1), SUBTRACT(2), MULTIPLY(3), DIVIDE(4);
+		
+		final int precedence;							//enumerator to store precedence of operators 
+		Operator(int p) {
+			precedence = p;
+		}
+	}
+	
+	private HashMap<String, Operator> operatorMap = new HashMap<String, Operator>(); {{
+		operatorMap.put("+", Operator.ADD);
+		operatorMap.put("-", Operator.SUBTRACT);
+		operatorMap.put("*", Operator.MULTIPLY);		//HashMap to associate string operators with their enumerator counterparts
+		operatorMap.put("/", Operator.DIVIDE);
+	}};
+	
+	private boolean isHigherPrec(String op, String sub) {
+		return (operatorMap.containsKey(sub) &&
+				operatorMap.get(sub).precedence >= operatorMap.get(op).precedence);		//method that when called checks precedence for given operator in
+	}																					//inFixToPostFix method
+	
+	public String infixToPostfix(String infix) {
+		StringBuilder postFix = new StringBuilder();	//StringBuilder object to store postFix expression as it is converted 
+		Stack<String> op = new Stack<String>();		//stack to store operators before they are added to StringBuilder object 
+		
+		for (String token : infix.split(" ")) {	
+			if (operatorMap.containsKey(token)) {		//checks if is operator
+				while (!op.isEmpty() && isHigherPrec(token, op.peek())) {
+					postFix.append(op.pop()).append(' ');					//checks precedence against mapping
+				}
+				op.push(token);
+			}
+			else if (token.equals("(")) {		//pushes opening bracket to stack
+				op.push(token);
+			}
+			else if (token.equals(")")) {
+				while (!op.peek().equals("(")) {			//if closing bracket is found pop operators to StringBuilder until 
+					postFix.append(op.pop()).append(' ');	//opening bracket is found, then discard.
+				}
+				op.pop();
+			}
+			else {
+				postFix.append(token).append(' ');
+			}
+		}
+		while(!op.isEmpty()) {
+			postFix.append(op.pop()).append(' ');			//pop rest of operators to StringBuilder 
+		}
+		return postFix.toString();
+	}
+	
+	public String calcPostFixLong(String postFix) {		//calculate result if it is a long
```

Comment:
There is much redundant code between the Long and double versions. You could probably have merged these and reduced the number of lines of code.